### PR TITLE
Added 2018 Projects!

### PR DESCRIPTION
**Pull request by vychun**
Added a markdown file that contains all of the projects of 2018!

### CODE REVIEW

File: 2018-projects.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+### [PaperStx](https://github.com/Jakobeha/PaperStx) ###
+
+#### Made by Jakob Hain ####
+
+An editor for block syntax, like the syntax in Scratch. It's a webpage/webapp whose custom logic is entirely client-side (e.g. you don't need a server to run it, you can just load the HTML file).
+The block syntax has basic types. Not every block can be connected to every other block. The blocks are colored, and they're have colored "holes" for other blocks - the only blocks which can go in a hole are the ones with one of the holes' colors.
+First, you create a template file. You follow a very simple DSL to either create your own block syntax or recreate an existing one (e.g. Scratch). Then, you upload this file to the webpage. Then, the webpage will give you all of the different block syntax forms, grouped by class. You can drag them onto a canvas, and create your block syntax.
+
+### [jgka](https://github.com/tylercote/spot-a-match) ###
```
Comment (applebynj) [inline 2018-projects.md:193]: should this be removed? im confused by it

File: 2018-projects.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+---
+
+### [Emotiquote](https://abigailhodge.github.io/emotiquote/) (Won Best Visual Award!) ###
+
+#### Made by Abigail Hodge, Michelle Pine, Sharon He, Sam Price ####
+
+Our interests in machine learning and web dev/UI prompted us to develop a web application which handles the Indico API and Mediawiki API to enable emotional analysis of very large and interesting datasets, namely, famous quotes (drawn from Wikiquotes).
+
+
+### [Nudge](https://github.com/sbernal2018/HackBeanPot2018) (Won Rookie of the Year Award and Genuine's #InventTogether Award!)###
```
Comment (applebynj) [inline 2018-projects.md]: trailing ### remove

Comment (applebynj) [reply 2018-projects.md]: or add space rather before ###

File: 2018-projects.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+#### Made by Prisca Joseph, Sarah Wessel, Rucha Khanolkar, and Megan Lau ####
+
+CourseOn is a web application to help students to organize which courses they need to take throughout their time at Northeastern. This allows students to generate their grad plan easily, dynamically, and interactively.
+
+### [pupDate](https://github.com/annachoi4/pupDate) ###
+
+#### Made by Anna Choi, Andres Rodriguez, Gabriella Roman, and Christopher Trinh ####
+
+A web application for dog owners to find their dogs some nearby playmates, since Anna owns a dog and is always looking for playmates! Made in PyCharm.
```
Comment (applebynj) [inline 2018-projects.md:165]: somethings wrong here, desc displays as large header text

File: 2018-projects.md (hunk 1 above)
Comment (vychun) [reply 2018-projects.md:193]: i basically just copied and pasted the descriptions that were in the forms - a lot of them were really long so it's definitely possible to shorten them and just remove them. 

just out curiousity, what are you talking about in this case?

File: 2018-projects.md (hunk 3 above)
Comment (aceyoung9) [reply 2018-projects.md:165]: @applebynj Hi I still get these emails so I figured I'd give you a hint. Look at L166. H1's in MarkDown are also triggered by a line after of 3-or-so hyphens.

```md
example
----
```
turns to

example
---

Comment (vychun) [reply 2018-projects.md:165]: ahh thank you for catching that - i had put those in there to help differentiate between copy and sections i'll remove that!

**Pull request merged**